plugins/test/unit/__init__.py

```diff
@@ -9,6 +10,10 @@
 devel_base.block_load_conf()
 
 
+if django.VERSION >= (1, 9):
+    django.setup()
```

Comment:
This function has nothing specifically to do with unit tests. If it's needed for testing, it's likely needed for all of Django. Some indication about what this function is, and why we're calling it where we're calling it is needed.